packages/firebase_auth/example/android/app/build.gradle

```diff
@@ -54,6 +54,10 @@ flutter {
 }
 
 dependencies {
+androidTestImplementation 'androidx.test:runner:1.2.0'
+androidTestImplementation 'androidx.test:rules:1.2.0'
+androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```

Comment:
Indentation is a little weird here, also these lines are redundant with the lines below.